include/tc/core/execution_engine.h

```diff
@@ -105,6 +106,11 @@ class ExecutionEngine {
  protected:
   size_t emplaceExecutor(std::unique_ptr<ExecutorInfo> p);
 
+  size_t getHandle(
+      const std::string& name,
+      const std::vector<const DLTensor*>& inputsInfo,
+      const MappingOptions& options);
```

Comment:
`MappingOptions` are CUDA-specific (have grid, block sizes and most configs are GPU-related).  I used a string for options in the parent classes...

Reply:
I know you're just copying code, but it would be nice if you could add some
documentation to this function (before or after you moved it).
As to `MappingOptions` being CUDA-specific, perhaps it should be renamed
or subclassed from a generic version (assuming there are any options that
make sense in general).

Reply:
yes that's the proper way to go after we start having a CPU mapper and can properly write / test different backends.
In the meantime I'll use string here to avoid immediate breakage in the upcoming CPU mapper.

Reply:
:+1: for string